apps/web/src/pages/ApplicantDashboardPage/components/HeroCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link } from "@gc-digital-talent/ui";
+import { ChevronDoubleRightIcon } from "@heroicons/react/24/solid";
+import * as React from "react";
+
+export type Color =
+  | "primary"
+  | "secondary"
+  | "tertiary"
+  | "quaternary"
+  | "quinary";
+
+export const colorMap: Record<Color, Record<string, string>> = {
+  primary: {
+    "data-h2-background-color":
+      "base:all(primary.light) base:dark:iap(primary)",
+    "data-h2-color":
+      "base:all(black) base:all:admin(white) base:all:iap(white)",
+  },
+  secondary: {
+    "data-h2-background-color":
+      "base(secondary) base:dark:admin(secondary.lighter) base:dark:iap(secondary.light)",
+    "data-h2-color":
+      "base:all(black) base:all:admin(white) base:all:iap(white)",
+  },
+  tertiary: {
+    "data-h2-background-color":
+      "base(tertiary.lighter) base:dark:iap(tertiary.light)",
+    "data-h2-color": "base:all(black) base:all:iap(white)",
+  },
+  quaternary: {
+    "data-h2-background-color":
+      "base(quaternary) base:dark:iap(quaternary.light)",
+    "data-h2-color": "base:all(black) base:all:iap(white)",
+  },
+  quinary: {
+    "data-h2-background-color": "base(quinary) base:dark:iap(quinary.light)",
+    "data-h2-color": "base:all(black) base:all:iap(white)",
+  },
+};
+
+export interface HeroCardProps {
+  color: Color;
+  title: string;
+  href: string;
+  children?: React.ReactNode;
+  ariaRole?: React.AriaRole;
+}
+
+export const HeroCard = ({
+  color,
+  title,
+  href,
+  children,
+  ariaRole,
+}: HeroCardProps) => {
+  return (
+    <div
+      data-h2-background-color="base:all(white)"
+      data-h2-radius="base(s)"
+      data-h2-shadow="base(s)"
+      data-h2-flex-grow="base(1)"
+      role={ariaRole}
```

Comment:
If we use `role=navigation` here, we should also provide either `aria-labelledby` or `aria-label` to [distinguish this navigation from other ones](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#accessibility_concerns:~:text=If%20there%20is%20more%20than%20one%20navigation%20landmark%20role%20or%20%3Cnav%3E%20element%20in%20a%20document%2C%20provide%20a%20label%20for%20each%20landmark.%20This%20label%20will%20allow%20an%20assistive%20technology%20user%20to%20be%20able%20to%20quickly%20understand%20the%20purpose%20of%20each%20landmark.).

This does open a whole new can of worms 🪱 since, if possible it is recommended to use the `nav` over `role=navigation` if possible as well, that `<nav>` should contain either a `<ul>` or `<ol>` (in this case a `<ul>` would be appropriate.

Though annoying (sorry 😓 ), I don't think this should be too difficult.

```tsx
export interface HeroCardProps {
  ...
  asNav?: boolean;
}

export const HeroCard = ({ asNav, ... }) => {
  const Wrapper = asNav ? "nav" : "div";
  
  return (<Wrapper>...</Wrapper>);
}
```

Then it would be just updating the `HeroCardProfileItem` to be `<li>` inside a `<ul>`.

Reply:
OK, I think I implemented what you described.  :sweat_smile: I left the "resume" block as a regular div since it doesn't contain a collection of links.
ref - [switch to nav element and list](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5857/commits/6fd112f641549380c877f5bcc2f69d3f95586db8)